homeassistant/components/brottsplatskartan/sensor.py

```diff
@@ -56,12 +62,19 @@ def update(self) -> None:
         """Update device state."""
 
         incident_counts: defaultdict[str, int] = defaultdict(int)
-        incidents = self._brottsplatskartan.get_incidents()
+        get_incidents: dict[str, list] | Literal[
```

Comment:
If the library would support typing by including a `py.typed` file in its package we wouldn't need to type the return value here explicitly.

Reply:
The typing in the library seems to be incorrect too. It says `Union[list, bool]` but actually returns bool or a dict.

Reply:
I know. I add it for myself when I refactor as it makes it easier to track doing the changes.
But I'll make an upstream PR to add it (and also some other changes)

Reply:
Returning `False` for errors is also not a good practice. The library should raise library specific exceptions.

Reply:
Let me just point out I didn't make the library nor did the integration 😁 
I just added the config flow at some point and now just bump the library to catch up

Reply:
Yeah I know. I see the last commit in the library was four years ago.